### PR TITLE
Set DPI for new view

### PR DESCRIPTION
**Pull request by rojkov**
This PR depends on https://github.com/tmeshkova/gecko-dev/pull/84

### CODE REVIEW

File: src/qopenglwebpage.cpp [hunk 1]
```diff
@@ -81,6 +83,7 @@ void QOpenGLWebPage::createView()
         d->mContext->GetApp()->SetIsAccelerated(true);
         d->mView = d->mContext->GetApp()->CreateView(mParentID, mPrivateMode);
         d->mView->SetListener(d);
+        d->mView->SetDPI(QGuiApplication::primaryScreen()->logicalDotsPerInch());
```
Comment (rainemak) [inline src/qopenglwebpage.cpp]: I still think that this should be ```physicalDotsPerInch``` and "apz.touch_start_tolerance" pref adjusted accordingly.

Thinking also this one:

```
AsyncPanZoomController::GetTouchStartTolerance()
{
  return (gfxPrefs::APZTouchStartTolerance() * APZCTreeManager::GetDPI());
}
```

Comment (rainemak) [inline src/qopenglwebpage.cpp]: In addition, logical is used for font scaling.

**Comment by rojkov** [comment]
Changed `logicalDotsPerInch` to `physicalDotsPerinch`.

I was testing `apz.touch_start_tolerance` with browser run from console with logical DPI equal to physical, so the value should be about right.

**Comment by rojkov** [comment]
Cherry picked to nemomobile

**Pull request merged**